build.gradle.kts

```diff
@@ -24,7 +24,6 @@ repositories {
     maven("https://oss.sonatype.org/content/repositories/snapshots")
 }
 
-val cloudVersion: String by project
```

Comment:
This is just personal preference but since Cloud makes use of multiple libraries (3) I find it best to keep a variable of the version to prevent duplicating it.